libraries/lib-sample-track/MixerSource.cpp

```diff
@@ -290,25 +336,23 @@ MixerSource::MixerSource(const SampleTrack &leader, size_t bufferSize,
    , mEnvelope{ options.envelope }
    , mMayThrow{ mayThrow }
    , mTimesAndSpeed{ move(pTimesAndSpeed) }
-   , mInputTrack( mnChannels )
-   , mSamplePos( mnChannels )
+   , mInputSequence( mnChannels )
    , mSampleQueue{ initVector<float>(mnChannels, sQueueMaxLen) }
-   , mQueueStart( mnChannels, 0 )
-   , mQueueLen( mnChannels, 0 )
+   , mQueueStart{ 0 }
+   , mQueueLen{ 0 }
    , mResampleParameters{ highQuality, leader, rate, options }
    , mResample( mnChannels )
    , mEnvValues( std::max(sQueueMaxLen, bufferSize) )
    , mpMap{ pMap }
 {
    size_t j = 0;
    for (auto channel : TrackList::Channels(&leader))
-      mInputTrack[j++].SetTrack(channel->SharedPointer<const SampleTrack>());
+      mInputSequence[j++]
+         .SetSequence(channel->SharedPointer<const SampleTrack>());
```

Comment:
`SharedPointer<const SampleFrameSequence>()` ?

Reply:
That would not compile.  Anyway, this line (changed in the first commit) is ultimately changed again, at "MixerSource uses SampleTrackCache "widely" and doesn't use Track..."